packages/p2p/src/components/buy-sell/buy-sell.jsx

```diff
@@ -0,0 +1,112 @@
+import React              from 'react';
+import PropTypes          from 'prop-types';
+import {
+    Button,
+    Dialog,
+    Loading }         from 'deriv-components';
+// import { WS } from '../../utils/websocket';
+import                         './buy-sell.scss';
+import Popup              from './popup.jsx';
+
+class BuySell extends React.Component {
+    state = {
+        buy_sell_list: [],
+        show_popup   : false,
+        type         : 'buy',
+        selected_ad  : {},
+        is_loading   : true,
+    }
+
+    componentDidMount() {
+        // Call API to retrieve buy_sell ad list
+        this.setState({
+            buy_sell_list: [
+                ...this.state.buy_sell_list,
+                {
+                    id             : 'buy_id',
+                    advertiser     : 'John doe',
+                    country        : 'Indonesia',
+                    currency       : 'IDR',
+                    type           : 'buy',
+                    asset          : 'USD',
+                    fix_price      : 12000,
+                    amount         : 20,
+                    min_transaction: 15000,
+                    max_transaction: 240000,
+                    payment_method : 'Bank transfer',
+                    advertiser_note: 'Whatsapp: +60182655318 please send to Maybank 239847238947 JOHN DOE',
+                },
+                {
+                    id             : 'sell_id',
+                    advertiser     : 'John doe',
+                    country        : 'Indonesia',
+                    currency       : 'IDR',
+                    type           : 'sell',
+                    asset          : 'USD',
+                    fix_price      : 12000,
+                    amount         : 20,
+                    min_transaction: 15000,
+                    max_transaction: 240000,
+                    payment_method : 'Bank transfer',
+                    advertiser_note: 'please contact through Whatsapp: +60182655318',
+                },
+            ],
+            is_loading: false,
+        });
+    }
+
+    handleClick = (ad) => {
+        this.setState({ selected_ad: ad, show_popup: true });
+    }
+
+    onConfirmClick = () => {
+        // TODO: [p2p] will create order from API
```

Comment:
The point of TODO tags are to make them searchable, I don't think the tag `[p2p]` is very safe here, what do you think?